### PR TITLE
[WIP] [AKS] Vendor and enable 2020-11-01 api

### PR DESCRIPTION
**Pull request by bcho**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2020_11_01/models/_models.py [hunk 1]
```diff
@@ -0,0 +1,2604 @@
...
+        'power_state': {'key': 'properties.powerState', 'type': 'PowerState'},
+        'availability_zones': {'key': 'properties.availabilityZones', 'type': '[str]'},
+        'enable_node_public_ip': {'key': 'properties.enableNodePublicIP', 'type': 'bool'},
+        'scale_set_priority': {'key': 'properties.scaleSetPriority', 'type': 'str'},
+        'scale_set_eviction_policy': {'key': 'properties.scaleSetEvictionPolicy', 'type': 'str'},
+        'spot_max_price': {'key': 'properties.spotMaxPrice', 'type': 'float'},
+        'tags': {'key': 'properties.tags', 'type': '{str}'},
+        'node_labels': {'key': 'properties.nodeLabels', 'type': '{str}'},
+        'node_taints': {'key': 'properties.nodeTaints', 'type': '[str]'},
+        'proximity_placement_group_id': {'key': 'properties.proximityPlacementGroupID', 'type': 'str'},
```
Comment (bowang-666) [inline src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2020_11_01/models/_models.py:245]: I don't see my properties here, is this not generated based on latest dev branch?

**Comment by bcho** [comment]
Yep it’s using the one before your pr 😂 I will close it for newer version

**Comment by bcho** [comment]
This PR is based on older version of 1101, we should vendor newer generated one cc @paulgmiller @bowang-666

**Comment by yonzhan** [comment]
AKS

File: src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2020_11_01/models/_models.py (hunk 1 above)
Comment (bcho) [reply src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2020_11_01/models/_models.py:253]: I updated the sdk from the latest generated artifacts, should have your changes now!

**Comment by bcho** [comment]
close for #2681 2681

**Pull request closed without merging**